services/crypto_delete.proto

```diff
@@ -29,18 +36,41 @@ option java_multiple_files = true;
 import "basic_types.proto";
 
 /**
- * Mark an account as deleted, moving all its current hbars to another account. It will remain in
- * the ledger, marked as deleted, until it expires. Transfers into it a deleted account fail. But a
- * deleted account can still have its expiration extended in the normal way.
+ * Delete an account.<br/>
+ * This will mark an account deleted, and transfer all tokens to a "sweep"
```

Comment:
Not sure exactly what is meant by a "sweep" account. Just another account that will receive all tokens and balances when this account is marked as deleted?

Reply:
A sweep account is a common term in banking and investment for any account that is intended to receive all of the "leftovers" from closing an account or a position.
The `transferAccountID` identifies the sweep account for this transaction.